loader/loader_linux.c

```diff
@@ -249,7 +249,9 @@ VkResult linux_read_sorted_physical_devices(struct loader_instance *inst, uint32
     }
     memset(sorted_device_info, 0, inst->total_gpu_count * sizeof(struct LinuxSortedDeviceInfo));
 
-    loader_log(inst, VULKAN_LOADER_INFO_BIT | VULKAN_LOADER_DRIVER_BIT, 0, "linux_read_sorted_physical_devices:  Original order:");
+    loader_log(inst, VULKAN_LOADER_INFO_BIT | VULKAN_LOADER_DRIVER_BIT, 0,
+               "linux_read_sorted_physical_devices: (App Version %d.%d)", inst->app_api_major_version, inst->app_api_minor_version);
```

Comment:
Seems a bit odd to print out the app version here. Maybe this should be apart of the CreateInstance call? Though if there are multiple instances, i'd be unclear which instance is using which version. Definitely am thinking out loud.

Reply:
I was just trying to make this info available, but I'll remove it for now.  It's just a comment after all.